<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>571</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>端口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="serialCb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baudrateCb">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>24</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="databitCb">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>24</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="checkbitCb">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stopbitCb">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>540</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>接收设置</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>166</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="recvTS">
        <property name="text">
         <string>接收时间戳</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="ClearBt">
        <property name="text">
         <string>清空接收</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="chk_rev_hex">
        <property name="text">
         <string>Hex接收</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chk_rev_line">
        <property name="text">
         <string>自动换行</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>发送设置</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QCheckBox" name="chk_send_hex">
        <property name="text">
         <string>Hex发送</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="sendTS">
        <property name="text">
         <string>发送时间戳</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="txtSendMs">
        <property name="maximum">
         <number>3600000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="autoSend">
        <property name="text">
         <string>自动发送/ms</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="recvEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Microsoft YaHei UI'; font-size:5pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>540</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBt">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>600</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Closed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearSend">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>640</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>清空发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="serialStatusLb">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>450</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenBt">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QChartView" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>961</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openChart">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QLabel" name="openChartLb">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口示波器</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
